pkg/operator/configobservation/node/observe_latency_profile.go

```diff
@@ -0,0 +1,127 @@
+package node
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/configobservation"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	configV1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+var notReadyTolerationSecondsPath = []string{"apiServerArguments", "default-not-ready-toleration-seconds"}
+var unreachableTolerationSecondsPath = []string{"apiServerArguments", "default-unreachable-toleration-seconds"}
```

Comment:
What happens on a downgrade? I think nothing will overwrite previously set values. Could you please double-check it?

Reply:
In case of a downgrade this config observer, latency controller as well as config node CRD object will be absent; the absence of config observer would ensure that the argument gets unset? unset is default value.